### DISCUSSION TITLE
Filesystem URI format

### DISCUSSION THREAD

**Discussion by ColobusDev** [music-assistant/support#961] [Q&A] [answered]
What's the format for adding folders/albums/playlists to the queue from the filesystem provider?

I've got spotify working in automations using the mass.queue_command service with uri of both "spotify:album:2WT1pbYjLJciAR26yMebkH" and "spotify://album/2WT1pbYjLJciAR26yMebkH", but I haven't been able to figure out what uri I need for local files

Note, I'm able to play/queue up local albums from the MA screen, but any URIs I've tried in the service call give an "Invalid URI" error

**Comment by OzGav** [comment, marked as answer]
Please refer to the FAQ https://github.com/music-assistant/hass-music-assistant/discussions/710 In short you just use the name of the album and MA works it out by querying the database

**Comment by OzGav** [comment]
Relates to v1
